deb/build

```diff
@@ -44,15 +69,29 @@ fi
 libversion=`echo $version_number | grep -o -P -e "([0-9]+\.?)+$"`
 lib_major=$(echo $libversion | cut -f1 -d .)
 
+# create temp file to write changelog to
 changelog=`mktemp`
 trap "rm -f '$changelog'; exit 1" INT TERM QUIT
 
-so_libname="./install/usr/lib/$so_filename"
+# (re)create symbolic links with the distributed package name, dist_pkgname
+cd $libdir
+touch $dist_so_libname.$libversion
+ln -sf $dist_so_libname.$libversion $dist_so_libname.$lib_major
+ln -sf $dist_so_libname.$lib_major $dist_so_libname
```

Comment:
Yeah. It would change the linking to -ldrizzle--redux6 for versions major branch 6.x.x.  Some libraries have use major-minor to distinguish the api level of the library, e.g. `somelib-2.0.so.1.0.1` but given that semver doesn't break ABI compatibility I reckon major version is sufficient.